src/Resources/contao/controllers/BackendMain.php

```diff
@@ -203,6 +205,20 @@ protected function output()
 			$this->Template->websiteTitle = \Config::get('websiteTitle');
 		}
 
+		/** @var AuthorizationCheckerInterface $authorizationChecker */
+		$authorizationChecker = \System::getContainer()->get('security.authorization_checker');
+
+		/** @var RouterInterface $router */
+		$router = \System::getContainer()->get('router');
+
+		$logoutLink = $router->generate('contao_backend_logout');
+
+		if ($authorizationChecker->isGranted('ROLE_PREVIOUS_ADMIN')) {
+			$logoutLink = $router->generate('contao_backend', [
+				'_switch_user' => '_exit',
```

Comment:
this is a new feature, right? Previously a login just logged out whatever user was currently in. Might be confusing if we don't change the logout label to "switch back to X".

Reply:
The current label is `Close the current session` and thus not wrong in my opinion.

Reply:
Changed in e2bed632d.